source/style-guide/style/punctuation.txt

```diff
@@ -60,6 +60,14 @@ Use the following guidelines for colons:
 - If a title or heading includes a colon, capitalize the first word
   that follows the colon, regardless of its part of speech.
 
+- "Lowercase or capital letter after a colon. When a colon is used within
+  a sentence, as in the first two examples in 6.61, the first word following
+  the colon is lowercased unless it is a proper noun.
+  When a colon introduces two or more sentences or when it introduces
+  speech in dialog or a quotation or question, the first word following
+  it is capitalized" (from Chicago Manual of Style 6.63, v.17).
```

Comment:
Question: Do we have a directive for block quotes?

Suggestion: Bold the same text as the Chicago Manual of Style.

```suggestion
- "**Lowercase or capital letter after a colon.** When a colon is used within
  a sentence, as in the first two examples in 6.61, the first word following
  the colon is lowercased unless it is a proper noun.
  When a colon introduces two or more sentences or when it introduces
  speech in dialog or a quotation or question, the first word following
  it is capitalized" (from Chicago Manual of Style 6.63, v.17).
```

Reply:
As mentioned in Slack, we have blockquote.
Paraphrased this so I think the quotation is no longer applicable.